test/checkpoint/workers.c

```diff
@@ -59,8 +59,8 @@ create_table(WT_SESSION *session, COOKIE *cookie)
           "memory_page_max=64KB,log=(enabled=false),%s",
           kf, vf, lsm));
     else
-        testutil_check(__wt_snprintf(config, sizeof(config),
-          "key_format=%s,value_format=%s,log=(enabled=false),%s", kf, vf, lsm));
+        testutil_check(
```

Comment:
This is a test/checkpoint test that is to verify the checkpoint on disk. Enabling the logging to the tables will defeat the purpose of the test? 

I agree to enable the logging as we found a data corruption issue during the test. Do we proceed with the above change and later revert it? Or add a new option to let the tables be logged?